fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClientMain.java

```diff
@@ -20,7 +20,8 @@ public static void main(String... args) throws Exception {
         properties.put(BulkDataConfigUtil.BATCH_USER, "fhiradmin");
         properties.put(BulkDataConfigUtil.BATCH_USER_PASS, "change-password");
                 
-        properties.put(BulkDataConfigUtil.BATCH_TRUSTSTORE, "/Users/paulbastide/git/wffh/wlp/wlp/usr/servers/defaultServer/resources/security/fhirTruststore.jks");
+        properties.put(BulkDataConfigUtil.BATCH_TRUSTSTORE,
+                "/Users/paulbastide/git/wffh/wlp/wlp/usr/servers/defaultServer/resources/security/fhirTruststore.p12");
```

Comment:
can it be a relative path?
if not, then maybe add a placeholder for git and a comment to say you should change it to the path on your system?

Reply:
Fixed